libnetwork/drivers/ipvlan/ipvlan_setup.go

```diff
@@ -42,6 +47,7 @@ func createIPVlan(containerIfName, parent, ipvlanMode string) (string, error) {
 			ParentIndex: parentLink.Attrs().Index,
 		},
 		Mode: mode,
+		Flag: flag,
```

Comment:
It seems like we are always setting this now.
Is there a behavior change here?

Reply:
No behavior change here. Bridge (0) is the traditional mode where slaves can cross-talk among
themselves apart from talking through the master device.